packages/elements/src/datetime-picker/index.ts

```diff
@@ -519,6 +502,10 @@ export class DatetimePicker extends ControlElement implements MultiValue {
       this.syncInputValues();
     }
 
+    if (changedProperties.has('placeholder')) {
```

Comment:
let's change default value of `format` prop to `'dd-MMM-yyyy'` as well. Plus, add a comment saying that "`format` is the default value of `placeholder`"

Reply:
Fixed

Reply:
still missing comment "`format` is the default value of `placeholder`"